dev/radar/docker-compose.yaml

```diff
@@ -0,0 +1,36 @@
+version: "3"
+services:
+  radar-mssql:
+    extends:
+      file: ../common-services.yaml
+      service: mssql
+    volumes:
+      - leaf-radar-mssql:/var/opt/mssql
+
+  radar-coreapi:
+    extends:
+      file: ../common-services.yaml
+      service: coreapi
+    volumes:
+      - ./keys:/.keys:ro
+      - ./appsettings.json:/app/API/appsettings.json:ro
+    environment:
+      LEAF_JWT_KEY_PW: ${RADAR_JWT_KEY_PW}
+      LEAF_APP_DB: Server=radar-mssql,1433;Database=LeafDB;uid=sa;Password=Th3PA55--8zz
+      LEAF_CLIN_DB: Server=clin-db,3306;Database=radar;uid=db-user;Password=cnics@CIRG;
```

Comment:
I understand why these are here in plaintext for the moment, but at some point we should discuss how best to store these credentials/parameters in a manner that is not publicly available.

Reply:
I would accept this in the short term if this was a private repository, but it isn't. The credentials really should go in a separate .env file that never hits version control, and be templated in like in `LEAF_JWT_KEY_PW` above if not passed in entirely.

Reply:
Access is already restricted in other ways (network isolation, shell access, strong password) that make directly accessing the DB server impractical. 

It's hard for me to imagine how the static password could be exploited, but would restricting DB server access (by IP) to only allow clients on the docker network be sufficient?

Security often comes at the cost of convenience, but adding an additional manual configuration step (for production) is easy enough to justify. I'll go ahead and remove the static password, unless the IP address restrictions are sufficient.

Reply:
The thing I'm worried about isn't so much having anyone connect to the DB server who shouldn't (but that is a concern), but that throwaway dev passwords tend to get reused. Keeping the password separate also makes it possible to quietly revoke and replace it later.